CHANGELOG.rst

```diff
@@ -1,5 +1,89 @@
 .. _changelog:
 
+0.7.0
+-----
+
+This release brings a lot of important new features to both Renkulab and the
+Renku CLI. It's our best one yet!
+
+New features
+~~~~~~~~~~~~
+
+- **Project templates**: you can now create custom templates for your projects and
+  use them on project initialization. Great for groups or courses! See the
+  `docs <https://renku.readthedocs.io/en/latest/user/templates.html>`_ for more
+  details.
+
+- **Datasets**: you can now search for datasets and import them into projects
+  directly from the Renkulab web UI. The dataset import and creation flow has
+  also been much improved thanks to changes in the core service backend.
+
+- **Data import**: files can be imported into datasets directly from a URL.
+
+- **Project migration**: the metadata of renku projects occassionaly changes. In
+  order for all the components to work well together, the metadata must be kept
+  in sync. Previously you needed to do this manually with the CLI but now it's
+  as easy as clicking a button.
+
+- **Template simplification**: We have decoupled the CLI version from the base image and
+  made it easier to override in your own environments. See the `project template
+  README <https://github.com/SwissDataScienceCenter/renku-project-template/blob/master/README.md>`_
+  for details.
+
+- **Improved editor**: All text editing components are using an enhanced editor
+  that allows seamless switching between WYSIWYG and markdown.
+
+
+- **``renku save``**: the Renku CLI now features a ``save`` command that
```

Comment:
Not sure if this gets rendered correctly with `**''`

![Screenshot_20201029_164146](https://user-images.githubusercontent.com/43481553/97597059-b512d100-1a05-11eb-855e-f56f86fc661e.png)